### DISCUSSION TITLE
Add PIX as a financial support to the apps.

### DISCUSSION THREAD

**Discussion by goliv04053** [FossifyOrg/General-Discussion#61] [Ideas]
I think it would be amazing if you added PIX as a way to give support to the project. I am Brazilian and most of us use PIX to transfer money. I think it would be awesome if you added that.

**Comment by inson1** [comment]
@goliv04053 btw why are you using it? why its better?

**Reply by goliv04053** [reply]
I'm Brazilian. PIX is an instant payment platform created and managed by the monetary authority of Brazil. There is a lot of Brazilians who use it. So, I think that for us and some people in LATAM who has business or projects next to Brazil (so they use it). Also, it's better because it's instant. I can send the money and the person will recieve it very fast. Instant.

**Comment by lucasmz-dev** [comment]
I'm Brazilian and Pix is probably one of the best payment methods out there when it comes to fiat, but I'm unsure how they would achieve this. They would need a trusted party in Brazil which would convert BRL to USD. (or maybe Monero / another cryptocurrency)

A workaround right now is to buy Monero (or one of the other accepted cryptos) and using it instead. That's how I donated to @Signal a while ago. (Though it was actually more annoying because I had to use SideShift.io to convert it to Litecoin cuz that's the only thing they accepted)

**Comment by naveensingh** [comment]
Hi!

I haven't done any research on this but maybe there's hope: https://stripe.com/docs/payments/pix. 
For now, the options on the [main page](https://github.com/FossifyOrg#support-fossify-heart) will do.
